src/features/onboarding/accountCreatePassphrase/AccountCreatePassphraseScreen.tsx

```diff
@@ -41,25 +41,31 @@ const AccountCreatePassphraseScreen = () => {
     setDisabled(false)
   }, [viewedWords])
 
-  const renderItem = ({ item, index }: { item: string; index: number }) => (
-    <Card
-      marginHorizontal="s"
-      variant="elevated"
-      flex={1}
-      overflow="hidden"
-      backgroundColor="white"
-      padding="l"
-      alignItems="center"
-      flexDirection="row"
-    >
-      <Text variant="h1" color="purpleLight" maxFontSizeMultiplier={1}>{`${
-        index + 1
-      }. `}</Text>
-      <Text variant="h1" color="purpleDark" maxFontSizeMultiplier={1}>
-        {upperFirst(item)}
-      </Text>
-    </Card>
-  )
+  const renderItem = ({ item, index }: { item: string; index: number }) => {
+    const isFirst = index === 0
+    const isLast = index + 1 === mnemonic?.words.length
+    return (
+      <Card
+        marginHorizontal="s"
+        marginLeft={isFirst ? 'l' : undefined}
+        marginRight={isLast ? 'l' : undefined}
+        variant="elevated"
+        flex={1}
+        overflow="hidden"
+        backgroundColor="white"
+        padding="l"
+        alignItems="center"
+        flexDirection="row"
+      >
+        <Text variant="h1" color="purpleLight" maxFontSizeMultiplier={1}>{`${
```

Comment:
The vertical alignment is off on these. This seems to fix it
```ts
        <Text
          variant="bold"
          fontSize={39}
          color="purpleLight"
          maxFontSizeMultiplier={1}
        >{`${index + 1}. `}</Text>
        <Text
          variant="bold"
          fontSize={39}
          color="purpleDark"
          maxFontSizeMultiplier={1}
        >
          {upperCase(item)}
        </Text>
```

Reply:
Weird bug haha, updated